curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64dc963d866fff3dd0329993.md

```diff
@@ -0,0 +1,39 @@
+---
+id: 64dc963d866fff3dd0329993
+title: Step 10
+challengeType: 20
+dashedName: step-10
+---
+
+# --description--
+
+Print the variable you created in the previous step and feel free to change the number of disks to see how fast the required minimum number of moves increases.
+
+# --hints--
+
+You should print the `number_of_moves` variable.
+
+```js
+({ test: () => assert.match(code, /print\s*\(\s*number_of_moves\s*\)/) })
+
+```
+
+# --seed--
+
+## --seed-contents--
+
+```py
+--fcc-editable-region--
+NUMBER_OF_DISKS = 3
+number_of_moves = 2 ** NUMBER_OF_DISKS - 1
```

Comment:
```suggestion
number_of_moves = 2**NUMBER_OF_DISKS - 1
```

I would suggest adding whitespace around the operator with the lowest priority (-), like [PEP 8](https://peps.python.org/pep-0008/#other-recommendations) suggests, and omit the spaces around the operator with the highest priority (**).

I suggest this for all the subsequent steps too.